packages/trader/src/App/Containers/populate-header.tsx

```diff
@@ -21,6 +27,9 @@ const PopulateHeader = observer(() => {
         p =>
             p.contract_info &&
             symbol === p.contract_info.underlying &&
+            //Added check for unsupported and forward starting contracts, which have not started yet
+            getSupportedContracts()[p.contract_info?.contract_type as keyof ReturnType<typeof getSupportedContracts>] &&
+            hasContractStarted(p.contract_info) &&
```

Comment:
This filter is getting too big. Can we abstract the filter into separate functions to make the code readability better?

Reply:
Sure, will apply for both files (mobile and desktop Recent Positions)